src/const.ts

```diff
@@ -0,0 +1,54 @@
+export const Settings = {
+  resultCount: 312
+};
```

Comment:
Чекни [критерий](https://up.htmlacademy.ru/react/15/criteries#b9) про енамки
```suggestion
export const enum Settings {
  ResultCount = 312
}
```